src/etc/inc/filter.inc

```diff
@@ -2313,7 +2313,9 @@ function filter_generate_user_rule_arr($rule) {
 	$line = filter_generate_user_rule($rule);
 	$ret['rule'] = $line;
 	$ret['interface'] = $rule['interface'];
-	if ($rule['descr'] != "" and $line != "") {
+	if ($rule['tracker'] != "" and $line != "") {
+		$ret['descr'] = "label \"" . fix_rule_label("USER_RULE: {$rule['tracker']}") . "\"";
```

Comment:
The idea of adding rule description to USER_RULE: label is to make it easy to identify what rule it is when you are reading /tmp/rules.debug. Since every rule has a tracker, it will replace all USER_RULE: labels by the tracker, what is not a good idea